README.md

```diff
@@ -14,3 +15,18 @@ filesizeParser('200KB')
 filesizeParser('200Kb')
 25600
 ```
+
+The default [base](https://en.wikipedia.org/wiki/Gigabyte) is `2`, you can use base `10`
```

Comment:
Could you update to point to [the base section](https://en.wikipedia.org/wiki/Gigabyte#Base_2_definition)?